### PR TITLE
JENKINS-67858 Place ionicons at the same height as their associated text

### PR DESCRIPTION
**Pull request by NotMyFault**
See [JENKINS-67858](https://issues.jenkins.io/browse/JENKINS-67858)

This is an alternative approach to https://github.com/jenkinsci/jenkins/pull/6304, that addresses all occasions of misaligned ionicons, also in plugins:

<details>
  <summary>Maven integration, 2.337</summary>

</details>

<details>
  <summary>Maven integration, proposed change</summary>

</details>

</details>

<details>
  <summary>Node overview, for reference</summary>

</details>

Tango and icons from other sources and in other locations, like the sidebar, are unaffected by this change.

Closes https://github.com/jenkinsci/jenkins/pull/6304

### Proposed changelog entries

* [JENKINS-67858](https://issues.jenkins.io/browse/JENKINS-67858): Place icons at the same height as their associated text (regression in 2.335).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Review by janfaracik — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]
Looks great, thanks for taking care of this!

(Untested.)

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

File: war/src/main/less/base/style.less
Comment (jtnord) [inline war/src/main/less/base/style.less:1570]: why top?  to me this seems like it should be middle?

see https://issues.jenkins.io/browse/JENKINS-68356  changing it to middle actually fixes that bug.  The screenshots in the PR description also look like they are aligned in the middle - and the generic icon is using middle -> https://github.com/jenkinsci/jenkins/pull/6341/files#diff-300f384d47e9aad264a350440de6adfe3cbb8a33c6925206700382807f950b6cR1609

**Pull request merged**
